<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;h2&gt; Калькулятор викидів забруднюваних речовин&lt;/span&gt;&lt;/h2&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>551</width>
      <height>349</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;h3&gt;Діоксид сірки SO2:&lt;/h3&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="formula1">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/formula.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;де Q&lt;span style=&quot; vertical-align:sub;&quot;&gt;i&lt;/span&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;r&lt;/span&gt; - нижча робоча теплота згоряння палива, МДж/кг;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Georgia','serif'; font-style:italic; color:#000000;&quot;&gt;S&lt;/span&gt;&lt;span style=&quot;color:#000000; vertical-align:super;&quot;&gt; r&lt;/span&gt;&lt;span style=&quot;color:#000000;&quot;&gt;– вміст сірки в паливі на робочу масу за проміжок часу &lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt;P&lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt;, %;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Symbol'; color:#000000;&quot;&gt;h&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; font-style:italic; color:#000000; vertical-align:sub;&quot;&gt;I&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000;&quot;&gt; – ефективність зв’язування сірки золою або &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000;&quot;&gt;сорбентом у енергетичній установці;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Symbol'; color:#000000;&quot;&gt;h&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; font-style:italic; color:#000000; vertical-align:sub;&quot;&gt;II&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000;&quot;&gt; – ефективність очистки димових газів від оксидів сірки;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Symbol'; color:#453427;&quot;&gt;b&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000;&quot;&gt;– коефіцієнт роботи сіркоочисної установки.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_5"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Результат:        </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Розрахувати</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>494</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Розрахувати</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>494</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:large; font-weight:600;&quot;&gt;Діоксид вуглецю CO2:&lt;/span&gt;&lt;/h3&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="formula1_2">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/img/formula2.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Georgia','serif'; font-style:italic; color:#000000;&quot;&gt;C&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; font-style:italic; color:#000000; vertical-align:super;&quot;&gt;r&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; font-style:italic; color:#000000;&quot;/&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000;&quot;&gt;- масовий вміст вуглецю в паливі на робочу масу, %;&lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Georgia','serif'; font-style:italic; color:#000000;&quot;&gt;Q&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; font-style:italic; color:#000000; vertical-align:sub;&quot;&gt;i&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; font-style:italic; color:#000000; vertical-align:super;&quot;&gt;r&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; font-style:italic; color:#000000;&quot;/&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000;&quot;&gt;- нижча робоча теплота згоряння палива, МДж/кг;&lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000; vertical-align:sub;&quot;&gt;C&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000;&quot;&gt; - ступінь окислення вуглецю палива (&lt;/span&gt;&lt;a href=&quot;http://eco.com.ua/content/vikidi-zabrudnyuvalnih-rechovin-u-atmosferu-vid-energetichnih-ustanovok-metodika#%D0%94%D0%BE%D0%B4%D0%B0%D1%82%D0%BE%D0%BA_%D0%90&quot;&gt;&lt;span style=&quot; font-family:'Georgia','serif'; text-decoration: underline; color:#000000;&quot;&gt;додаток А&lt;/span&gt;&lt;/a&gt;&lt;a name=&quot;_Hlt250817039&quot;/&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000;&quot;&gt;)&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000;&quot;&gt;;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Georgia','serif'; font-style:italic; color:#000000;&quot;&gt;k&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000; vertical-align:sub;&quot;&gt;C&lt;/span&gt;&lt;span style=&quot; font-family:'Georgia','serif'; color:#000000;&quot;&gt; - показник емісії вуглецю палива, г/ГДж.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>308</y>
      <width>494</width>
      <height>75</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Результат:        </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
